node/forest_libp2p/src/service.rs

```diff
@@ -356,6 +375,41 @@ where
     }
 }
 
+async fn bitswap_timeout_task(
+    outgoing_bitswap_query_ids: BitswapOutgoingQueryTable,
+    outgoing_bitswap_query_cancellation_tx: Sender<libp2p_bitswap::QueryId>,
+) {
+    loop {
+        let mut timeout_queries = vec![];
```

Comment:
Maybe just move outside the vector creation to avoid allocation?

Reply:
Fixed